### ISSUE TITLE
Add env overwrite for nested data

### ISSUE THREAD

**Issue by dijam** [dijam/good-config#1]
`nconf` has support for reading from env variables and translate them to conf keys. 
```
nconf.env({                                                                                                            
    separator: '__', // Two dashes
    lowerCase: true
});
```
info link: https://stackoverflow.com/questions/45434620/how-to-read-herokus-nested-process-env-vars-object-in-nconf

**Comment by dijam** [comment]
Fixed in version 1.1.0 via envify

**Comment by dijam** [comment]
Fixed in version 1.1.0 via envify
